create_from_yaml is converting rbac.authorization.k8s.io/v1 incorrectly

The create_from_yaml function in utils is gets an error when trying to create resources that use the rbac api.  For example, when trying to create the role below with create_from_yaml.
```
kind: Role
apiVersion: rbac.authorization.k8s.io/v1
metadata:
  namespace: default
  name: pod-reader
```
It results in the following error.
```
>           k8s_api = getattr(client, fcn_to_call)(k8s_client)
E           AttributeError: module 'kubernetes.client' has no attribute 'Rbac.authorizationV1Api'
```
rbac.authorization.k8s.io/v1 is being converted into Rbac.authorizationV1Api instead of RbacAuthorizationV1Api.  Additional logic is needed to convert this correctly.

--- a/kubernetes/e2e_test/test_utils.py
+++ b/kubernetes/e2e_test/test_utils.py
@@ -99,6 +99,20 @@ class TestUtils(unittest.TestCase):
         self.assertIsNotNone(nmsp)
         core_api.delete_namespace(name="development", body={})
 
+    def test_create_rbac_role_from_yaml(self):
+        """
+        Should be able to create an rbac role.
+        """
+        k8s_client = client.api_client.ApiClient(configuration=self.config)
+        utils.create_from_yaml(
+            k8s_client, self.path_prefix + "rbac-role.yaml")
+        rbac_api = client.RbacAuthorizationV1Api(k8s_client)
+        rbac_role = rbac_api.read_namespaced_role(
+            name="pod-reader", namespace="default")
+        self.assertIsNotNone(rbac_role)
+        rbac_api.delete_namespaced_role(
+            name="pod-reader", namespace="default", body={})
+
     def test_create_deployment_non_default_namespace_from_yaml(self):
         """
         Should be able to create a namespace "dep",

--- a/kubernetes/e2e_test/test_yaml/rbac-role.yaml
+++ b/kubernetes/e2e_test/test_yaml/rbac-role.yaml
@@ -1,0 +1,9 @@
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  namespace: default
+  name: pod-reader
+rules:
+- apiGroups: [""] # "" indicates the core API group
+  resources: ["pods"]
+  verbs: ["get", "watch", "list"]

--- a/kubernetes/utils/create_from_yaml.py
+++ b/kubernetes/utils/create_from_yaml.py
@@ -97,8 +97,10 @@ def create_from_yaml_single_item(
     # Take care for the case e.g. api_type is "apiextensions.k8s.io"
     # Only replace the last instance
     group = "".join(group.rsplit(".k8s.io", 1))
-    fcn_to_call = "{0}{1}Api".format(group.capitalize(),
-                                     version.capitalize())
+    # convert group name from DNS subdomain format to
+    # python class name convention
+    group = "".join(word.capitalize() for word in group.split('.'))
+    fcn_to_call = "{0}{1}Api".format(group, version.capitalize())
     k8s_api = getattr(client, fcn_to_call)(k8s_client)
     # Replace CamelCased action_type into snake_case
     kind = yml_object["kind"]

